docs/library/uasyncio.rst

```diff
@@ -266,3 +266,22 @@ Event Loop
 .. method:: Loop.close()
 
     Close the event loop.
+
+.. method:: Loop.set_exception_handler(handler)
+
+    Set the exception handler to call when a Task raises an exception that is not
+    caught.  The *handler* should accept two arguments: ``(loop, context)``.
+
+.. method:: Loop.get_exception_handler()
+
+    Get the current exception handler.  Returns the handler, or ``None`` if no
+    custom handler is set.
+
+.. method:: Loop.default_exception_handler(context)
```

Comment:
shouldn't that be `Loop.default_exception_handler(loop, context)`?

Reply:
Because it's a method there is an implicit first argument, self/loop.  It's a bit confusing when looking at the implementation (which uses class methods for efficiency), but this signature matches the CPython docs.

Reply:
I suspected that, I still find the docs function/method/classmethod/etc distinctions very difficult to spot/understand... Here it's particularly confusing because in the previous paragraph it says "the *handler* should accept two arguments" and then there's the default handler which visibly only accepts one... Oh well, another puzzle for the reader to solve...

Reply:
Actually @tve is right. The current implementation is not a classmethod and you need to pass the loop explicitly. 
```
loop.default_exception_handler({})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: default_exception_handler() takes 2 positional arguments but 1 were given
>>> loop.default_exception_handler(loop,{})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kevin/.micropython/lib/uasyncio/core.py", line 254, in default_exception_handler
KeyError: message
```

However, the docs still are correct. The default_exception_handler is called without the loop while the custom exception handler needs to provide a loop argument. But that's how it is in CPython. (But tbh, why would anyone call the default exceoption handler manually?)

Reply:
> (But tbh, why would anyone call the default exceoption handler manually?)

- it's confusing when reading the docs, so it made me wonder which is wrong since they seem to contradict each other
- if you want to reset to the default exception handler you want to pass that into set_exception_handler but the signature doesn't appear to match at first sight

Reply:
If you want to reset the exception handler, you call set_exception_handler(None).
So you don't have to worry about how the default_exception_handler is implemented.

The docs are correct (like Cpython) but the implementation of of default_exception_handler is not. It would need to be @classmethod